operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -528,25 +528,6 @@
             ||| % $._config,
           },
         },
-        {
```

Comment:
Did you mean to remove this one?

Reply:
😱 

oh my bad. just put that piece back in place.